tests/arti/internal/test_type_hints.py

```diff
@@ -106,15 +105,13 @@ def test_get_item_from_annotated(
         (list[int], Annotated[list[int], 5]),
         (str, Any),
         (str, Union[int, str]),
-        (tuple, typing.Tuple),
```

Comment:
Ruff converted `typing.Tuple` to just `tuple`, which is redundant with the other tests. They do sometimes have slightly different behavior at runtime, but not worth messing with.